corehq/apps/cloudcare/static/cloudcare/js/formplayer/menus/controller.js

```diff
@@ -120,6 +120,11 @@ hqDefine("cloudcare/js/formplayer/menus/controller", function () {
             menusUtils.showBreadcrumbs(menuResponse.breadcrumbs);
             if (!appPreview && ((menuResponse.langs && menuResponse.langs.length > 1 && changeFormLanguage) || enablePrintOption)) {
                 menusUtils.showFormMenu(menuResponse.langs, changeFormLanguage);
+                // Make menu the last element in the breadcrumbs
+                const formMenuEl = $("#form-menu");
+                const formMenuElParent = formMenuEl.parent();
+                formMenuEl.remove();
+                formMenuElParent.append(formMenuEl);
```

Comment:
I am confused for why we are doing this here and not in html itself?

Reply:
The element for the dropdown is already present in the html, breadcrumbs are dynamically added by the javascript. That causes the dropdown to be the second item in the list and not the last one which it should be. The code above removes the element from the 2nd spot and adds it to the end of the list.

Reply:
> The code above removes the element from the 2nd spot and adds it to the end of the list.

Relying on the order seems risky. Can we have a more reliable way?

> breadcrumbs are dynamically added by the javascript.

May be at this point we could ensure the order we need?

Reply:
Fixed. [e097bce](https://github.com/dimagi/commcare-hq/pull/32995/commits/e097bce5650d937780b841d8ced6fc2264d8f258)

![image](https://github.com/dimagi/commcare-hq/assets/31195143/e935ed40-b58b-4689-a868-b851218f0794)

Reply:
Thank you! I didn't review the new code closely, but I appreciate the change in direction, and from skimming it, it looks simpler.